core/src/main/java/org/mini2Dx/core/geom/Circle.java

```diff
@@ -29,7 +29,7 @@ public Circle(int radius) {
 		this.radius = radius;
 	}
 	
-	public Circle(float centerX, float centerY, int radius) {
```

Comment:
This is an important (and unrelated) change to the rest of the commit. When working with this, I noticed that the circle had an 'int' argument and a 'float' variable backing it up.